exporter.go

```diff
@@ -209,13 +218,14 @@ func (b *Exporter) Listen(e <-chan Events) {
 					}
 				}
 			} else {
+				eventsUnmapped.Inc()
 				metricName = escapeMetricName(event.MetricName())
 			}
 
 			switch event.(type) {
 			case *CounterEvent:
 				counter := b.Counters.Get(
-					metricName+"_counter",
+					b.suffix(metricName, "counter"),
```

Comment:
While we're fixing this, can we change this to `total` to match Prometheus metric name guidelines?